spec/integration/put_content/content_with_a_previously_published_edition_spec.rb

```diff
@@ -4,10 +4,21 @@
   include_context "PutContent call"
 
   before do
+    Timecop.freeze("2017-01-02 12:23")
+  end
+
+  after do
+    Timecop.return
+  end
+
+  before do
     stub_request(:put, %r{.*content-store.*/content/.*})
+    Timecop.freeze(Time.local(2017, 9, 1, 12, 0, 0))
```

Comment:
There seems to already be a `Timecop.freeze` above, do we need both?

Reply:
We don't need both - will remove one 👍